spec/features/notifications_spec.rb

```diff
@@ -3,7 +3,14 @@
 RSpec.describe "Notifications" do
   describe " visit notifications" do
     before (:each) { sign_in_as('Editor') }
-
+    let!(:notification)  { create(:notification, subject: "Howdy from Hopedale") }
+    context "basic CRUD" do
+      let!(:notification)  { create(:notification, subject: "Howdy from Hopedale") }
+      it "should display one at a time" do
+        visit notification_path(notification)
+        expect(page).to have_content("Howdy")
```

Comment:
this only proves that there is at least one Howdy. it doesn't prove that it only displays one at a time.

also the 2nd one should have a different variable name (`let!(:notification_2){...}`) so that no-one mistakenly thinks you're just overwriting the 1st one without creating 2 entries in the db, which you are.

also, this has nothing to do with the "label" (title) so should probably be in a separate pr